plone/app/content/tests/test_folder.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 from DateTime import DateTime
+from plone import api
```

Comment:
plone.api is not allowed in plone core code in order to avoid circular dependencies.